### PR TITLE
Improve UI Variables hide datasource and add dependency from prometheus monitored instances for ClickHouse_Queries_dashboard.json

### PR DESCRIPTION
**Pull request by Slach**
Improve UI for ClickHouse_Queries_dashboard.json
- set `basicAuth: false` and `usePost: true` by default for all installed ClickHouse datasources

Signed-off-by: Eugene Klimov <eklimov@altinity.com>

### CODE REVIEW

File: grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json [hunk 1]
```diff
@@ -3845,7 +3845,7 @@
         "allValue": null,
         "current": {},
         "datasource": "${DS_PROMETHEUS}",
-        "definition": "label_values(chi_clickhouse_metric_Uptime, hostname)",
+        "definition": "label_values(chi_clickhouse_metric_Uptime{exported_namespace=\"$exported_namespace\",chi=\"$chi\"}, hostname)",
```
Comment (Slach) [inline grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json:3848]: fix for prometheus metrics dashboard UI variables

File: grafana-dashboard/ClickHouse_Queries_dashboard.json [hunk 2]
```diff
@@ -870,21 +885,65 @@
...
         "includeAll": false,
         "label": "K8S ClickHouse installation",
         "multi": false,
-        "name": "chi",
+        "name": "db",
         "options": [],
         "query": "vertamedia-clickhouse-datasource",
         "refresh": 1,
-        "regex": "/k8s/",
+        "regex": "k8s-$exported_namespace-$chi",
```
Comment (Slach) [inline grafana-dashboard/ClickHouse_Queries_dashboard.json:899]: main part of fix
add dependency on prometheus label related variables
@amonakhov so created clickhouse cluster datasource, should follow this naming conversion

**Comment by alex-zaitsev** [comment]
What is the reason to have variable named 'exported_namespace' and not just a 'namespace'? Any name conflict?

**Comment by Slach** [comment]
@alex-zaitsev expected_namespace it same variable name which we use in `Main Prometheus` dashboard and it came from directly from `metrics_exporter`

**Pull request merged**